airgun/views/hostgroup.py

```diff
@@ -17,6 +17,18 @@
 from airgun.widgets import RadioGroup
 
 
+class ActivationKeyDropDown(ActionsDropdown):
+    dropdown = Text('.//*[self::div]')
+
+    @property
+    def items(self):
+        """Returns a list of all dropdown items as strings."""
+        self.dropdown.click()
```

Comment:
I think you should use `self.open()` here, to include the check against `is_open` before clicking the dropdown.

Reply:
The video helps immensely, I see what you mean.